mage_ai/orchestration/pipeline_scheduler.py

```diff
@@ -94,6 +94,14 @@ def schedule(self, block_runs: List[BlockRun] = None) -> None:
                         tags=merge_dict(tags, dict(metrics=self.pipeline_run.metrics)),
                     )
 
+                pipeline_schedule = PipelineSchedule.query.get(self.pipeline_run.pipeline_schedule_id)
+                if (pipeline_schedule is not None):
```

Comment:
you don't need the parentheses in python

Reply:
you can also combine the two if statements if you want

Reply:
Yeah it’ll evaluate the first statement first and if it fails it’ll just move on